extensions/standard-processors/processors/LogAttribute.h

```diff
@@ -33,24 +33,23 @@
 #include "core/PropertyType.h"
 #include "core/RelationshipDefinition.h"
 #include "core/logging/LoggerConfiguration.h"
-#include "utils/gsl.h"
 #include "utils/Export.h"
 
 namespace org::apache::nifi::minifi::processors {
 
 class LogAttribute : public core::Processor {
  public:
-  explicit LogAttribute(std::string_view name, const utils::Identifier& uuid = {})
+  explicit LogAttribute(const std::string_view name, const utils::Identifier& uuid = {})
       : Processor(name, uuid) {
     logger_->set_max_log_size(-1);
   }
   ~LogAttribute() override = default;
 
   EXTENSIONAPI static constexpr const char* Description = "Logs attributes of flow files in the MiNiFi application log.";
 
-  EXTENSIONAPI static constexpr auto LogLevel = core::PropertyDefinitionBuilder<5>::createProperty("Log Level")
+  EXTENSIONAPI static constexpr auto LogLevel = core::PropertyDefinitionBuilder<6>::createProperty("Log Level")
       .withDescription("The Log Level to use when logging the Attributes")
-      .withAllowedValues({"info", "trace", "error", "warn", "debug"})
+      .withAllowedValues({"trace", "debug", "info", "warn", "error", "critical"})
```

Comment:
Do we really need critical here? I'm not even sure about its usefulness elsewhere, but a LogAttribute message will certainly not be a critical error, which I consider those that kill or crash the minifi agent.

Do you prefer a consistent set of log levels, even if one of the options don't make sense to be user-exposed?

Reply:
I prefer consistency, it doesnt cost us anything. I can image a scenario where these logs are critical not from minifis point of view but rather from the flow's perspective. (e.g. you log out critically a failure relationship because its business critical to have the previous operation to be successfull)